### PR TITLE
Typographical Error Fixes

### PR DESCRIPTION
**Pull request by Kenner-G**

### CODE REVIEW

File: src/pocketmine/lang/Installer/en.ini [hunk 1]
```diff
@@ -26,20 +26,20 @@ op_who = OP player name (example, your game name)
...
+whitelist_warning = You will have to add the players to the white-list.
 
 query_warning1 = Query is a protocol used by different tools to get information of your server and players logged in.
 query_warning2 = If you disable it, you won't be able to use server lists.
 query_disable = Do you want to disable Query?
 rcon_info = RCON is a protocol to remote connect with the server console using a password.
 rcon_enable = Do you want to enable RCON?
 rcon_password = RCON password (you can change it later)
-usage_info = The anonymous usage data allows us to calculate global statistics for PocketMine-MP and its plugins. You can view them on stats.pocketmine.net
+usage_info = The anonymous usage data allows us to calculate global statistics for PocketMine-MP and it's plugins. You can view them on stats.pocketmine.net
```
Comment (dktapps) [inline src/pocketmine/lang/Installer/en.ini]: This is incorrect.

Comment (SOF3) [inline src/pocketmine/lang/Installer/en.ini]: It was correct. It was a possessive pronoun for "PocketMine-MP's plugins", not "it is plugins".

**Comment by dktapps** [comment]
Unneeded "corrections" and the grammatical corrections are incorrect.

File: src/pocketmine/lang/Installer/en.ini (hunk 1 above)
Comment (Muqsit) [inline src/pocketmine/lang/Installer/en.ini]: http://www.talkenglish.com/grammar/grammar.aspx

Comment (Muqsit) [inline src/pocketmine/lang/Installer/en.ini]: grammar nazis everywhere oh god 👆 
positive

Comment (DiegoGTRatty) [inline src/pocketmine/lang/Installer/en.ini]: Is this what the internet has come to? Grammar Nazis that don't even know how to spell themselves?

**Pull request closed without merging**